File: mario.py
from cs50 import get_int

row = get_int("Height: ")
# Check if row is between 1 to 8
while row < 1 or row > 8:
    print("Invalid, please enter again.")
    row = get_int("Height: ")
space = row - 1
for i in range(row):
    # print number of space base of row-1
    for j in range(space):
        print(" ", end="")
    # print number of hashtag base of the current row number
    for k in range(i+1):
        print("#", end="")
    print("  ", end="")
    # re-print number of hashtag agian
    for p in range(i+1):
        print("#", end="")
    print("")
    space -= 1

File: mario.c
#include "cs50.h"
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        // Getting user input of Pyramic Height
        height = get_int("Piramid Height: \n");
    }
    // Getting only the range of number - 1 to 8
    while (height < 1 || height > 8);

    // Adding block height that depend on input value
    int x = height;
    for (int i = 1; i <= x; i++)
    {
        for (int j = x - i; j > 0; j--)
        {
            printf(" ");
        }
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf(" ");
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}
